### DISCUSSION TITLE
Can't add users because the "license" can't be activated.

### DISCUSSION THREAD

**Discussion by Ilovemyhous** [photoprism/photoprism#4573] [Q&A]
Hello!
I have a friend who setup a PhotoPrism LXC container for me on Promox, which works, pictures can be uploaded, they get indexed, and everything.
I also have a PhotoPrism podman pod/container on my own server, which is version: 240711-2197af848-Linux-AMD64-Plus

Both instances are linked to my PhotoPrism account, which has an Essential subscirption.

From here on, I'll be talking on the LXC instance running on my friends server.
The problems with this, is that while I have the UI which tells me that I have Essential, and I see said instance in my PhotoPrism account, the CLI tells me otherwise. Even after restarting the LXC container.
![image](https://github.com/user-attachments/assets/bd414f4a-a76b-4d84-8a38-fa3313c25817)

But even is I create an account as admin, I can't login with that account, but the default admin account works.
![image](https://github.com/user-attachments/assets/7a7ebcd0-fb63-45ef-83d8-22fdce2e55ce)

Meanwhile, on my other instance, which, I know, isn't the same version, notr the same container type, everything just works fine.

I've already looked [here](https://github.com/photoprism/photoprism/discussions/3813), and [here](https://github.com/photoprism/photoprism/issues/1103), without much success though.

**Comment by graciousgrey** [comment]
As for the failed login, I would recommend checking the login attempts:

- https://docs.photoprism.app/user-guide/users/cli/#viewing-login-attempts

You can find our full troubleshooting checklist for failed logins here:

- https://docs.photoprism.app/getting-started/troubleshooting/#cannot-log-in

Please note that there is no official LXC image provided by us. So unfortunately we can't help much with LXC specific problems.

- https://docs.photoprism.app/getting-started/faq/#lxc-images

**Comment by Ilovemyhous** [comment]
After looking at [this](https://github.com/photoprism/photoprism/discussions/4247) discussion, I actually had the same problem... little did I know it solved the "license" problem.

But I still have problems where users wouldn't get logged in.
It turns out, I didn't had the config files, and apparently, it didn't read the .env file.
There was also an issue when I started the frontend from the terminal/daemon, and with systemd.

If anyone might also have this issue, this is what I put in my `/etc/systemd/system/photoprism.service`

```
[Unit]
Description=PhotoPrism service
After=network.target

[Service]
Type=simple
user=root
Environment="PHOTOPRISM_AUTH_MODE=password"
Environment="PHOTOPRISM_ADMIN_PASSWORD=XXXXXXXX"
Environment="PHOTOPRISM_HTTP_HOST=0.0.0.0"
Environment="PHOTOPRISM_HTTP_PORT=2342"
# Environment="PHOTOPRISM_SITE_CAPTION=Debug"
Environment="PHOTOPRISM_ORIGINALS_PATH=/opt/photoprism/photos/originals"
Environment="PHOTOPRISM_IMPORT_PATH=/opt/photoprism/photos/import"
ExecStart=/usr/local/bin/photoprism start
Restart=always

[Install]
WantedBy=multi-user.target
```
